### ISSUE TITLE
Faculty member adds individual sakai tools from /apps to give students more direct access to learning materials

### ISSUE THREAD

**Issue by profmikegreene** [DukeLearningInnovation/learn.duke#56]

**Comment by JustinJohnsen** [comment]
This user story is a bit vague to me. In what way is the access more direct? Is this describing a disaggregation of the Sakai functions into separate apps in learn.duke?

**Comment by profmikegreene** [comment]
Yes, kind of related to #62. The hypothesis being (IMO) that there are tools in the LMS that would be useful to (and save us from building our own versions) learning communities even if they don't use the LMS wholesale. 

Sakai's web content tool is an interesting example, we could use that for #39, even if faculty don't want to use Sakai in any other way for their course.
